opus/application/static_media/js/browse.js

```diff
@@ -1819,7 +1820,7 @@ var o_browse = {
                 galleryHtml += '<div class="op-modal-overlay">';
                 galleryHtml += '<p class="content-text"><i class="fas fa-binoculars fa-4x text-info" aria-hidden="true"></i></p>';
                 galleryHtml += '</div></a>';
-                galleryHtml += `<div class="op-last-modal-overlay text-success op-hide-element op-browse-gallery-tooltip" title="Last viewed in slideshow mode"></div>`;
+                galleryHtml += `<div class="op-last-modal-overlay text-success op-hide-element op-browse-gallery-bino-tooltip" title="Last viewed in slideshow mode"></div>`;
```

Comment:
minor comment - I had to hunt to figure out what "bino" referred to; might want to consider changing it to "binocular" or "binoculars"